### DISCUSSION TITLE
Syntax typing

### DISCUSSION THREAD

**Discussion by rockingrohit9639** [monkeytypegame/monkeytype#1714] [Ideas] [answered]
What about typing on the code syntax for different languages. Like instead of just simple text we can have code samples for typing.
It is just an idea, please add on your views.

It is the idea of one of my friends, so all credit to him.

**Comment by Miodec** [comment, marked as answer]
If you select a code language, and quote mode (and there are quotes available for selected language) you can test your speed at typing code.
